website/pages/blog/creating-a-new-js-plugin.mdx

```diff
@@ -0,0 +1,520 @@
+---
+title: "Creading a New JavaScript Source Plugin from Scratch"
+tag: product
+date: 2023/10/1
+author: michal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+In this blog post, I will walk you through the steps to create a new source plugin using CloudQuery JavaScript SDK. The purpose is to give you a basic overview of how to create tables and columns and how to sync the actual data. 
+
+## What you need to know
```

Comment:
```suggestion
## What You Need to Know
```

We should use consistent casing (we have `Getting Started` below and also `Plugin configuration`). Either uppercase only the first letter or use [title casing](https://en.wikipedia.org/wiki/Capitalization#Title_case)